macOS/DuckDuckGo/Updates/UpdateUserDriver.swift

```diff
@@ -261,7 +268,14 @@ final class UpdateUserDriver: NSObject, SPUUserDriver {
             Logger.updates.log("Updater dismissing obsolete update")
         }
 
-        if checkpoint == .restart && !ignoresCheckpoint {
+        guard autoRestartAllowed else {
+            onResuming = { reply(.install) }
+            updateProgress = .updateCycleDone(.pausedAtRestartCheckpoint)
+            Logger.updates.log("Updater paused at restart checkpoint")
+            return
+        }
+
+        if areAutomaticUpdatesEnabled || !autoRestartAllowed {
```

Comment:
Won’t `!autoRestartAllowed` always be `false` here following the guard statement?